wire-library/wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -966,6 +1079,18 @@ class KotlinGenerator private constructor(
             }
           })
         }
+        for (oneOf in boxOneOfs) {
+          val fieldName = localNameAllocator[oneOf]
+          add("if (%N != null) ", fieldName)
+          addStatement("%N += %P", resultName, buildCodeBlock {
+            add(fieldName)
+            // TODO(Benoit) Do we redact if one of the field is redacted?
```

Comment:
Do we have a tracking bug for these oneofs? Useful to put this in the checklist on that bug.